web/src/lib/utils/resources.ts

```diff
@@ -253,28 +253,30 @@ export async function getStudentsFromGroup(
     throw error(400, { message: 'Invalid student group id' });
   }
   const studentGroup = await getStudentGroup(additionalId);
-  const dataItems: DataItem[] = studentGroup.students.map(
-    (student): DataItem => ({
-      id: student.id,
-      firstName: student.firstName,
-      lastName: student.lastName,
-      tags: student.tags.map(({ name }) => name),
-    }),
-  );
+  const dataItems: DataItem[] = studentGroup.students
+    ? studentGroup.students.map(
+        (student): DataItem => ({
+          id: student.id,
+          firstName: student.firstName,
+          lastName: student.lastName,
+          tags: student.tags.map(({ name }) => name),
+        }),
+      )
+    : [];
   return {
     data: dataItems,
-    totalElements: Number(studentGroup.students.length),
+    totalElements: Number(studentGroup.students ? studentGroup.students.length : 0),
```

Comment:
Why does this need to be converted to a number? `studentGroup.students` should already be a number right? Also, you probably want `students?.length ?? 0`